docs/advanced-usage/user-quotas/index.md

```diff
@@ -0,0 +1,144 @@
+# User quotas
+
+Since
+[REANA 0.8.0](https://blog.reana.io/posts/2021/release-0.8.0/#cpu-and-disk-quota-accounting),
+it is possible to limit the amount of disk space and CPU time available to
+execute workflows with the help of user quotas.
+
+## Check usage and quota limits
+
+User quotas are optional features of REANA which might not be enabled in your
+cluster. To check if they are enabled and which are the usage limits, you can
+use the `quota-show` command of the `reana-client` command-line tool.
+
+First of all, you can list all the available resources:
+
+```console
+$ reana-client quota-show --resources
+cpu
+disk
+```
+
+Then, you can get more details about a given resource:
+
+```console
+$ reana-client quota-show --resource disk --report limit
+No limit.
+$ reana-client quota-show --resource disk --report usage
+192512
+$ reana-client quota-show --resource disk
+192512 used
+```
+
+In particular, you can select which resource to consider (`cpu` or `disk`) with
+the `--resource` option and you can choose to show only the usage or the quota
+limit of the given resource with `--report`.
+
+You can also specify the `-h`/`--human-readable` command-line option to display
+the disk or CPU usage in a human-readable format:
+
+```console
+$ reana-client quota-show --resource cpu --report limit -h
+1h
+$ reana-client quota-show --resource cpu --report usage -h
+1m 19s
+$ reana-client quota-show --resource cpu -h
+1m 19s out of 1h used (2%)
+```
+
+The usage of the resources is also displayed on the profile page of the web UI,
+which you can access by clicking on the person-shaped icon in the top-right
+corner of the web interface:
+
+!["Your quota" section of the profile page](/images/ui-profile-your-quota.png)
+
+!!! note
+    If the _"Your quota"_ section is not present, it means that user quotas
+    are not enabled in your cluster!
+
+## What to do when quotas are exceeded
+
+If you exceed your user quota of one of the resources, you will not be able to
+execute any new workflow until the resource usage will be back under the limit.
+In particular, if you have exceeded your CPU quota, there is nothing you can do
+except asking the administrators of your REANA instance to increase your CPU
+quota limit. However, if you have consumed all the available disk space, you can
+free up some of it by either deleting some past workflows or by removing some of
+their files.
+
+### Find workflows consuming the most disk space
+
+The easiest way to find out which workflows are using the most disk space is
```

Comment:
(Not sure about "easiest", it could depend on the user! Also, in the section above on this page, the CLI was presented before Web UI, so we may want to use same order here perhaps for consistency. (Or we could invert the order also above to have Web UI before CLI UI.) I assume we'd keep the former, hence suggested changes below.)

Using command line, you can use `reana-client list` command to also include the disk usage numbers for each workflow run by means of `--include-workspace-size` option, and have the list sorted by the most disk-space-consuming workflows first:

```console
$ reana-client list --include-workspace-size --sort size
NAME                      RUN_NUMBER   CREATED               STARTED               ENDED                 STATUS     SIZE
reana-demo-root6-roofit   2            2022-05-30T12:50:46   2022-05-30T12:50:58   2022-05-30T12:51:13   finished   180224
reana-demo-helloworld     1            2022-05-30T11:59:48   2022-05-30T12:00:00   2022-05-30T12:00:15   finished   12288
reana-demo-root6-roofit   1            2022-05-30T11:59:58   2022-05-30T12:00:09
```

Using web interface, you can go to the REANA home page and select...

Reply:
Each section now shows the instructions for the CLI before the ones for the web interface, with some little changes to the old text and to the proposed changes